lib/ansible/modules/packaging/os/yum.py

```diff
@@ -146,6 +150,22 @@
         The disabled plugins will not persist beyond the transaction.
     required: false
     version_added: "2.5"
+  releasever:
```

Comment:
It makes me a little nervous that we're not adding tests for new features.

Reply:
Yeah, I should really add more tests. The problem is that I'm not sure how best to test this just yet. Let me think about it a little and try to get something added that has a reasonable assertion.

Reply:
However, we don't have test coverage of that functionality for the `yum` module and I brought over roughly a dozen new options to `dnf` from `yum`.... but I do agree we still need test coverage. I'll look at this when I'm back.